lib/automaticupgrades/config.go

```diff
@@ -78,10 +80,12 @@ func GetUpgraderVersion(ctx context.Context) string {
 		out, err := exec.CommandContext(ctx, teleportUpgradeScript, "version").Output()
 		if err != nil {
 			log.WithError(err).Debug("Failed to exec /usr/sbin/teleport-upgrade version command.")
-		} else {
-			if version := strings.TrimSpace(string(out)); version != "" {
-				return version
-			}
+			return ""
+		}
+		ver, err := version.EnsureSemver(string(bytes.TrimSpace(out)))
+		if err != nil {
+			log.WithError(err).Debug("Unexpected teleport-upgrade version.")
+			return ver
 		}
 	}
 	return os.Getenv(EnvUpgraderVersion)
```

Comment:
Maybe if there's a broken version we should return "" or "(unspecified)", and we should return the actual version otherwise? What's the point of checking the binary if we're going to return the version of the upgrader at the time of first launch no matter what (assuming Teleport gets reloaded every time)?

Reply:
Oops, missing return